.gitignore

```diff
@@ -96,3 +96,8 @@ quarkus/data/*.db
 # Horreum #
 ###########
 provision/environment_data.json
+
+# OpenTofu / Terraform
+**/*.tfstate*
+**/*.terraform*
+**/.terraform.lock*
```

Comment:
IMHO we should keep the `.terraform.lock.hcl` files in the Git repository, as it prevents accidental / unintended updates of providers. 

Once that file exists, only a `opentofu init -upgrade` would upgrade to newer versions as allowed by the constraints given in the configurations.